### ISSUE TITLE
Add setting to disable the "* People are talking!" alert in browser tab

### ISSUE THREAD

**Issue by alxxlc** [prawnsalad/KiwiIRC#713]
It would be nice to have the ability to decide what types of notifications I receive when things happen in a chat. I enjoy the notification when my username is mentioned but I find it annoying to always have the "* People are talking!" flashing in my browser tab. People are always talking in the chat room that I lurk on so this is not something I really need to know about.

What I am requesting is just to have a check box in the settings that would allow me to disable the flashing tab indicator for people talking in a chat room but still get notifications when my username is mentioned. That would make Kiwi IRC the perfect client for my situation if this was implemented.

I know this is not a feature that everyone would want to use but it would be very nice to have. Thanks!

**Comment by NoorKhan** [comment]
I agree... if there's a custom style sheet solution to get around this I'd appreciate that

**Comment by prawnsalad** [comment]
We now have a re-organised settings window to allow more settings such as this. http://i.imgur.com/zjEkrXA.png

I think the window title alerts option would better suit in here now.
